src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -24,6 +28,11 @@ internal void InitializeLicense(string licenseText)
 
             if (string.IsNullOrWhiteSpace(licenseText))
             {
+                if (FirstTimeUser() && (Debugger.IsAttached && SystemInformation.UserInteractive))
+                {
+                    TrackFirstTimeUsageEvent();
+                    SetupFirstTimeRegistryKeys();
```

Comment:
@timbussmann @danielmarbach - Yes, the registry key should be created. I will make this code more clearer. The web service call will only be invoked if attached to the debugger, but the registry key will be created for a first time user. I'll make the change and then ping you.